janusgraph-dist/README.md

```diff
@@ -58,3 +58,12 @@ To build the .deb and .rpm packages:
 
 To delete the packaging scripts from the root of the repository, run
 `mvn -N -Ppkg-tools clean` from the janusgraph-dist module.
+
+## Upgrade cassandra-server version 
+
+Following files have to be update, if you update cassandra server version 
+in the default JanusGraph distributable. These files contains just small changes 
+which allows to include cassandra into our distributable.
```

Comment:
Following files have to be update**d**, if you update cassandra server version in the default JanusGraph **distribution**. These files contains just small changes which allows to include cassandra into our **distribution**.